code/modules/mob_spawn/ghost_roles/spider_roles.dm

```diff
@@ -168,4 +168,5 @@
 	if(QDELETED(src) || QDELETED(user) || !chosen_spider)
 		return FALSE
 	mob_type = chosen_spider
+	mob_name = "[mob_name] ([rand(1, 1000)])"
```

Comment:
need to set the mob's real_name var too. 

we have a helper that handles this. should be using either the unique_name var or living/set_name() if possible